tests/unit/utils.spec.js

```diff
@@ -53,3 +53,17 @@ describe('Test toggle element from list', () => {
     }
   );
 });
+
+describe('Test convert date to UTC', () => {
+  const date = new Date('Wed, 15 Jun 2020 07:00:00 GMT');
```

Comment:
Create a date as Europe/Paris and convert it to UTC and verify you have the wanted result.

Reply:
I think I have that in the test?, I am creating a date with timeZone 'Europe/Paris', convert it to UTC and check if the difference is 2 hours, however, I am not sure if the implementation is right.